### DISCUSSION TITLE
LFS installed and tracking but still not pushing large files

### DISCUSSION THREAD

**Discussion by AntonioMarcoRodrigo** [community/community#23830] [New to GitHub]
<p>Hello, </p>
<p>I need to push a Unity project, but one of my files is a Unity scene (.unity) and is 250mb. I already installed LFS and tracked this file:</p>
<pre><code>git lfs track "*.unity"

git add .gitattributes
</code></pre>
<p>But when I try to push, i get the same error i got before doing it:</p>
<table><tbody><tr><td><div><pre><span>$ git push </span>
<span>Uploading LFS objects: 100% (36/36), 273 MB | 0 B/s, done </span>
<span>Enumerating objects: 192, done. </span>
<span>Counting objects: 100% (192/192), done. </span>
<span>Delta compression using up to 4 threads </span>
<span>Compressing objects: 100% (105/105), done. </span>
<span>Writing objects: 100% (106/106), 56.70 MiB | 3.28 MiB/s, done. </span>
<span>Total 106 (delta 56), reused 0 (delta 0) </span>
<span>remote: Resolving deltas: 100% (56/56), completed with 50 local objects.</span>
<span>remote: </span><span>error</span><span>: GH001: Large files detected. You may want to try Git Large File S</span>
<span>torage - https://git-lfs.github.com.</span>
<span>remote: </span><span>error</span><span>: Trace: f8294b41f796452a459f39193e37e22f</span>
<span>remote: </span><span>error</span><span>: See http://git.io/iEPt8g for more information.</span>
<span>remote: </span><span>error</span><span>: File Assets/Scenes/Full.unity is 246.08 MB; this exceeds GitHub's</span>
<span> file size limit of 100.00 MB</span>
<span>To https://github.com/EXHOUL/Metalself.git </span>
 <span>! [remote rejected]</span><span> develop -&gt; develop (pre-receive hook declined) </span>
<span>error: failed to push some refs to 'https://github.com/EXHOUL/Metalself.git'</span> </pre></div></td></tr></tbody></table>
<p>Is there something I’m missing? Thanks in advance :slight_smile:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/antoniomarcorodrigo">@antoniomarcorodrigo</a>,</p>
<p>Thank you for being here! The error you’re seeing generally indicates that the file was already committed to the regular Git repository before the LFS tracking was added for it. I recommend reviewing our <a href="https://help.github.com/articles/moving-a-file-in-your-repository-to-git-large-file-storage/" rel="nofollow noopener">Moving a file in your repository to Git Large File Storage</a> guide for steps on moving these files over. Specifically the information within the blue box showing the command and information on using <code>git lfs migrate</code> should help you out here.</p>

**Comment by xiaohangliuai** [comment]
Try this:
  $ git checkout –-orphan new_branch    
  $ git add -A    
  $ git commit -am “Initial commit”    
  $ git branch -D main    
  $ git branch -m main    
  $ git push -f origin main
